### PR TITLE
Update redirects in Readme

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,15 +204,15 @@ Looking for resources for code languages? Take a look at [the awesome collection
...
 * [PrimerPro](https://github.com/sillsdev/PrimerPro) The purpose of PrimerPro is to assist the literacy worker in the development of primers for a given language. 
 * [pyannotation ![GitHub stars](https://img.shields.io/github/stars/cidles/pyannotation.svg)](https://github.com/cidles/pyannotation)  PyAnnotation is a Python Library to access and manipulate linguistically annotated corpus files.
-* [PyAnnotationGraph ![GitHub stars](https://img.shields.io/github/stars/PhonologicalCorpusTools/PyAnnotationGraph.svg)](https://github.com/PhonologicalCorpusTools/PyAnnotationGraph)  Implement the formal model for linguistic annotations described in Bird and Liberman (2001) using Python and SQL
-* [pyDelphin ![GitHub stars](https://img.shields.io/github/stars/goodmami/pydelphin.svg)](https://github.com/goodmami/pydelphin)  Python libraries for DELPH-IN (Friendly Fork)
+* [PyAnnotationGraph ![GitHub stars](https://img.shields.io/github/stars/PhonologicalCorpusTools/PyAnnotationGraph.svg)](https://github.com/PhonologicalCorpusTools/PolyglotDB)  Implement the formal model for linguistic annotations described in Bird and Liberman (2001) using Python and SQL
```
Comment (RichardLitt) [inline README.md]: This one isn't accurate. I don't think the original PyAnnotationGraph is here anymore - but PolyglotDB isn't the same repo, and should be added independently, while PyAnnotationGraph should be removed.

File: README.md [hunk 2]
```diff
@@ -323,14 +323,14 @@ Looking for resources for code languages? Take a look at [the awesome collection
...
 * [AndroidFieldDB ![GitHub stars](https://img.shields.io/github/stars/FieldDB/AndroidFieldDB.svg)](https://github.com/FieldDB/AndroidFieldDB)  An Android app which lets the user build a custom visual and auditory vocabulary, useful for guided anomia treatment and self designed language lessons by heritage speakers.
 * [AndroidFieldDBElicitationRecorder ![GitHub stars](https://img.shields.io/github/stars/FieldDB/AndroidFieldDBElicitationRecorder.svg)](https://github.com/FieldDB/AndroidFieldDBElicitationRecorder)  A general purpose video recording tool
-* [AndroidLanguageLearningClientForFieldDB ![GitHub stars](https://img.shields.io/github/stars/FieldDB/AndroidLanguageLearningClientForFieldDB.svg)](https://github.com/FieldDB/AndroidLanguageLearningClientForFieldDB)  An Android language learning app which plugs into a FieldDB corpus to create language learning apps.
+* [AndroidLanguageLearningClientForFieldDB ![GitHub stars](https://img.shields.io/github/stars/FieldDB/AndroidLanguageLearningClientForFieldDB.svg)](https://github.com/FieldDB/AndroidLanguageLessons)  An Android language learning app which plugs into a FieldDB corpus to create language learning apps.
```
Comment (RichardLitt) [inline README.md]: The description and name of this should be modified to fit. 

> Lets heritage speakers create self designed language lessons. https://play.google.com/store/apps/details?id=com.github.opensourcefieldlinguistics.fielddb.lessons.georgian

**Comment by RichardLitt** [comment]
@ReadmeCritic thanks for running your bot through this. I've made a few suggested changes. I'm not sure if you'll fix this manually, or if you can update your bot somehow to catch these? The changes might be repo specific.

File: README.md (hunk 2 above)
Comment (RichardLitt) [inline README.md]: As should the Github shields link.

**Comment by ReadmeCritic** [comment]
Will update manually :sweat_smile:

**Comment by ReadmeCritic** [comment]
@RichardLitt ready

**Comment by RichardLitt** [comment]
@ReadmeCritic Thank you! This was über awesome. Good work.

**Pull request merged**